lib/services/composer.js

```diff
@@ -50,6 +50,18 @@ function resolveComponentReferences(data, locals, filter = referenceProperty, ur
 
             return bluebird.reject(error);
           });
+        })
+        .catch(function (error) {
+          if (label.includes('content') && !ref.includes('@published')) {
+            log('info', `URI ${ref} not found, creating...`);
```

Comment:
I don't know if it hits the `log('error'` on line 48, but if it does we might want to change that to a warning when it is a missing component error since we are handling it.

Reply:
Yeah in this case it doesn't - warn would be better.